pkg/repl/client.go

```diff
@@ -31,7 +31,12 @@ const (
 	// Since on some of our Aurora tables with out-of-cache workloads only copy ~300 rows per second,
 	// we probably shouldn't set this any larger than about 1K. It will also use
 	// multiple-flush-threads, which should help it group commit and still be fast.
+	// This is only used as an initial starting value. It will auto-scale based on the DefaultTargetBatchTime.
 	DefaultBatchSize = 1000
+
+	// DefaultBatchTime is the target time for flushing REPLACE/DELETE statements.
```

Comment:
comment should say `DefaultTargetBatchTime`?

Reply:
Yes